_posts/2025-08-12-socratic-seminar-30.md

```diff
@@ -32,5 +32,8 @@ This meetup will be run in the format of a Socratic Seminar, inspired by other [
  - Introductions
  - Socratic Seminar Questions
  - Discussion
+ - # Agenda
+   - Tor dropping OS spoofing: What to know and effects [Watch the overview](https://www.youtube.com/watch?v=3wlNemFwbwE) joeFi
+ - Discussion
```

Comment:
Just for future reference, we usually delete the "Agenda will be determined closer to date" and add links there. But just so I dont forget I will merge as is